platforms-serverless/lambda-node-16/run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -eux
```

Comment:
```suggestion
set -euox pipefail
```

Reply:
Could you make this an issue maybe?

Because I don't think this should be done in this PR, if it's useful we could actually do it in all the bash scripts we have.